scePthreadAttrSetstack implementation

StevenMiller123 opened the pull request:
This function is used by Final Fantasy XV. If there are any other Pthread functions I should implement in this PR, please let me know.

Commit:
scePthreadAttrSetstack implementation

Used by Final Fantasy XV

psucien commented on src/core/libraries/kernel/thread_management.cpp:
```diff
@@ -1086,6 +1086,16 @@ int PS4_SYSV_ABI scePthreadAttrGetstack(ScePthreadAttr* attr, void** addr, size_
 }
 
+int PS4_SYSV_ABI scePthreadAttrSetstack(ScePthreadAttr* attr, void* addr, size_t size) {
+    int result = pthread_attr_setstack(&(*attr)->pth_attr, addr, size);
```
Before calling this function, we need to ensure that:
* `attr` is a valid pointer
* `stack_addr` is a valid pointer
* `size` is less than specified by PS4 system maximum

psucien commented on src/core/libraries/kernel/thread_management.cpp:
```diff
@@ -1086,6 +1086,16 @@ int PS4_SYSV_ABI scePthreadAttrGetstack(ScePthreadAttr* attr, void** addr, size_
+    if (result == 0) {
+        return SCE_OK;
+    }
+    return SCE_KERNEL_ERROR_EINVAL;
```
we use `ORBIS_` prefix there

Commit:
Address Comments

Verify parameters before calling the pthread_attr_setstack function.
Swap uses of SCE prefix with ORBIS prefix.

Commit:
Quick fix

Addresses the newest review and appears to fix issues caused in games by my previous commit.

psucien reviewed (approved):
Thank you!
